apps/embed-iframe/src/imported/style/theme.ts

```diff
@@ -44,34 +44,109 @@ const sizes = {
 const theme = extendTheme({
   components: {
     Text: {
-      baseStyle: {
+      baseStyle: (props: StyleFunctionProps) => ({
         fontWeight: "400",
         fontFamily: "Inter",
-      },
+        color: mode(light.grey[900], dark.grey[900])(props),
+      }),
       sizes,
     },
     Heading: {
-      baseStyle: {
+      baseStyle: (props: StyleFunctionProps) => ({
         fontWeight: "600",
         fontFamily: "Inter",
-      },
+        color: mode(light.grey[900], dark.grey[900])(props),
+      }),
       sizes,
     },
-    Button: buttonTheme,
-    Modal: modalTheme,
+    Button: {
+      sizes: {
+        md: { height: "38px", fontSize: "14px", minWidth: "38px" },
+        lg: { height: "48px", fontSize: "16px", minWidth: "48px" },
+      },
+      variants: {
+        primary: (props: StyleFunctionProps) => ({
+          bg: mode(light.grey.white, dark.grey.white)(props),
+          border: "1.5px solid",
+          borderColor: mode(light.grey[100], dark.grey[100])(props),
+          _hover: {
+            border: "1.5px solid",
+            borderColor: mode(light.grey[100], dark.grey[100])(props),
+            bg: mode(light.grey[100], dark.grey[100])(props),
+          },
+        }),
+      },
+      defaultProps: {
+        variant: "primary",
+        size: "md",
+      },
+    },
+    Modal: {
+      baseStyle: (props: StyleFunctionProps) => ({
+        dialog: {
+          paddingTop: "36px",
```

Comment:
May be better to write as `padding: "36px 42px"`?

Reply:
Good catch - fixed 🙂